packages/react/src/recipes/seo/Article/Article.module.css

```diff
@@ -28,13 +15,166 @@
+}
+
+.lightHero .hero {
+  background: var(--base-color-scale-white-0);
+}
+
+.darkHero .hero {
+  background: var(--base-color-scale-black-0);
+}
+
+.parallax {
+  position: absolute;
+  top: calc(-1 * var(--parallax-start-spacing));
+  width: 100%;
+  height: var(--hero-height);
+}
+
+.background,
+.foreground {
+  width: 100%;
+  height: 100%;
+
+  animation-fill-mode: forwards;
+  animation-timing-function: linear;
+  animation-timeline: scroll();
+  animation-range: 0vh var(--hero-height);
+}
+
+.background {
+  position: absolute;
+}
+
+.foreground {
+  position: relative;
 }
 
 .heroImage {
-  display: block;
+  min-width: 100%;
+  min-height: 100%;
+}
+
+.heroImageOverlay {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.lightHero .heroImageOverlay {
+  background: linear-gradient(
+    0deg,
+    rgba(255, 255, 255, 0.8) 0%,
+    rgba(255, 255, 255, 0) 50%,
+    rgba(255, 255, 255, 0.6) 100%
+  );
+}
+
+.darkHero .heroImageOverlay {
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.8) 0%, rgba(0, 0, 0, 0) 50%, rgba(0, 0, 0, 0.6) 100%);
+}
+
+.heroImageOverlayDark {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.8) 0%, rgba(0, 0, 0, 0) 50%, rgba(0, 0, 0, 0.6) 100%);
+}
+
+.foregroundContents {
+  padding: 0;
+  padding-block-start: calc(var(--base-size-20) + var(--parallax-start-spacing));
+  height: 100%;
+}
+
+.articleContents {
+  border-radius: var(--brand-borderRadius-xlarge);
+  margin-top: calc(-1 * var(--brand-borderRadius-xlarge));
+  background-color: var(--brand-color-canvas-default);
+  position: relative;
+  z-index: 90;
+}
+
+.lightHero .articleContents {
+  box-shadow: 0px -2px 49px 0px rgba(0, 0, 0, 0.25);
+}
+
+@keyframes parallax-background {
+  from {
+    top: 0;
+    filter: blur(0px);
+  }
+  to {
+    top: calc(var(--hero-height) * -0.4);
+    filter: blur(10px) brightness(0%);
+  }
+}
+
+@keyframes parallax-foreground {
+  from {
+    top: 0;
+  }
+  to {
+    top: calc(var(--hero-height) * -0.7);
+    filter: blur(2px) brightness(0%);
+  }
+}
+
+/* Progressive enhancement opt-in */
+@supports (animation-timeline: scroll()) {
+  @media (prefers-reduced-motion: no-preference) {
+    .parallax {
+      position: fixed;
+      top: 0;
+    }
+
+    .foreground {
+      animation-name: parallax-foreground;
+    }
+
+    .background {
+      animation-name: parallax-background;
+    }
+  }
+}
+
+@keyframes wipe {
+  from {
+    transform: scaleX(1);
+  }
+  to {
+    transform: scaleX(0);
+  }
+}
+
+.heroLine,
+.heroLineAnimated {
+  width: 100%;
+  min-height: 1lh;
+  position: relative;
+}
+
+.heroLineAnimated::after {
+  content: '';
+  display: inline;
+  position: absolute;
+  top: 0;
+  right: 0;
   width: 100%;
+  height: 100%;
+  background-color: var(--brand-color-accent-primary);
+  transform-origin: right;
+  animation: wipe 800ms forwards var(--brand-animation-easing-default);
```

Comment:
Non-blocking.. but we have an `--brand-animation-duration-extended` token at .7s that might be relevant for you here. It's 100ms faster though, but not sure how we landed at 800ms either.

Reply:
That works for me. I'll update it and ping @nsolerieu with a link to make sure he's happy